src/components/FormFooter.tsx

```diff
@@ -0,0 +1,80 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  filter: string,
+  setFilter: (filter: string) => void;
+  onClearCompleted: () => void,
+};
+
+export const FormFooter: React.FC<Props> = ({
+  todos, filter, setFilter, onClearCompleted,
+}) => {
+  const completedTodos = todos.filter(todo => todo.completed);
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${(todos.filter(todo => !todo.completed)).length} items left`}
```

Comment:
must be moved to a separate variable, you should not do such calculations on render